continuous_integration/recipe/run_test.py

```diff
@@ -13,19 +13,21 @@
 
 df = pd.DataFrame({"name": ["Alice", "Bob", "Chris"] * 100, "x": list(range(300))})
 ddf = dd.from_pandas(df, npartitions=10)
-c.create_table("my_data", ddf)
 
-got = c.sql(
-    """
-    SELECT
-        my_data.name,
-        SUM(my_data.x) AS "S"
-    FROM
-        my_data
-    GROUP BY
-        my_data.name
-"""
-)
-expect = pd.DataFrame({"name": ["Alice", "Bob", "Chris"], "S": [14850, 14950, 15050]})
+# This needs to be temprarily disabled since this query requires features that are not yet implemented
```

Comment:
Out of curiosity what functionality is missing for this to work?

Reply:
The `SUM()` operation has not yet been implemented as part of the `RexCall -> call.py` port